src/commands.rs

```diff
@@ -69,25 +67,18 @@ impl CommonOptions {
 
     /// Creates the warg client to use.
     pub fn create_client(&self, config: &Config) -> Result<FileSystemClient, ClientError> {
-        let client = match FileSystemClient::try_new_with_config(
-            self.registry.as_deref(),
-            config,
-            self.auth_token(config)?,
-        )? {
-            StorageLockResult::Acquired(client) => Ok(client),
-            StorageLockResult::NotAcquired(path) => {
-                println!(
-                    "blocking on lock for directory `{path}`...",
-                    path = path.display()
-                );
+        let client =
+            match FileSystemClient::try_new_with_config(self.registry.as_deref(), config, None)? {
```

Comment:
Was it intentional that you're now passing `None` for the auth token (and that you deleted the now-unused `auth_token` method below)?

Reply:
Yes, I was trying to simplify the burden on external crates getting the correct `auth_token`. So `None` gets the correct one, https://github.com/bytecodealliance/registry/blob/main/crates/client/src/lib.rs#L1403-L1405

Also, introduced `new_with_default_config()` to simplify, https://github.com/bytecodealliance/registry/blob/main/crates/client/src/lib.rs#L1428-L1430